app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -52,6 +51,8 @@ fun OtherUserProfileScreen(viewModel: OtherUserProfileScreenViewModel = hiltView
         onSendConnectionRequest = { viewModel.sendConnectionRequest() },
         onOpenConversation = { viewModel.openConversation() },
         onCancelConnectionRequest = { viewModel.cancelConnectionRequest() },
+        ignoreConnectionRequest = { viewModel.ignoreConnectionRequest() },
+        acceptConnectionRequest = { viewModel.acceptConnectionRequest() },
```

Comment:
Maybe, if possible and not too time demanding, we could change this to method reference, since @trOnk12 discovered a nice performance improvement, between calling lambdas vs method references

`ignoreConnectionRequest = viewModel::acceptConnectionRequest`

Reply:
done